TrustModel.md

```diff
@@ -0,0 +1,106 @@
+# Trust Model for Confidential Containers
+A clear definition of trust for the confidential containers project is needed to ensure the 
+components and architecture deliver the security principles expected for cloud native 
+confidential computing. It provides the solid foundations and unifying security principles 
+against which we can assess architecture and implementation ideas and discussions.
+
+The [Trust Modeling for Security Architecture Development article](https://www.informit.com/articles/article.aspx?p=31546) 
+defines Trust Modeling as :
+
+    A trust model identifies the specific mechanisms that are necessary to respond to a specific 
+    threat profile.
+
+    A trust model must include implicit or explicit validation of an entity's identity or the 
+    characteristics necessary for a particular event or transaction to occur.
```

Comment:
Nit: You could use a block quote for these quotes:

> A trust model identifies the specific mechanisms that are necessary to respond to a specific 
> threat profile.

> A trust model must include implicit or explicit validation of an entity's identity or the 
> characteristics necessary for a particular event or transaction to occur.

```suggestion
> A trust model identifies the specific mechanisms that are necessary to respond to a specific 
> threat profile.

> A trust model must include implicit or explicit validation of an entity's identity or the 
> characteristics necessary for a particular event or transaction to occur.
```

Reply:
Also, where do we plan to list the approach(es) taken for entity validation? Can this be linked to or a "TODO: See GitHub issue $url" added?

Reply:
Added the blockquotes and added Explicit validation of an entity's identity (attestation) to the Issue #28